README.txt

```diff
@@ -109,6 +109,11 @@ If the recurring opportunity will have the same volunteers each time, we’d rec
 
 == Changelog ==
 
+= 1.4.6 =
+* Adjust how the Volunteer Opportunities block is registered to prevent it from being registered on Volunteer Opportunity posts, the Widgets admin page, and the Widgets area in the customizer.
```

Comment:
How about updating this to be, "Adjusted the Volunteer Opportunities block registration so it can't be added to Volunteer Opportunities, or as a widget on the Widgets admin page or in the customizer."

Reply:
Updated.